Code/BuildSystem/CMake/ezUtils.cmake

```diff
@@ -111,19 +111,19 @@ function(ez_set_target_output_dirs TARGET_NAME LIB_OUTPUT_DIR DLL_OUTPUT_DIR)
 
 	set_target_properties(${TARGET_NAME} PROPERTIES
 		RUNTIME_OUTPUT_DIRECTORY_${EZ_BUILDTYPENAME_DEBUG_UPPER} "${OUTPUT_DLL_DEBUG}"
-		LIBRARY_OUTPUT_DIRECTORY_${EZ_BUILDTYPENAME_DEBUG_UPPER} "${OUTPUT_LIB_DEBUG}"
+		LIBRARY_OUTPUT_DIRECTORY_${EZ_BUILDTYPENAME_DEBUG_UPPER} "${OUTPUT_DLL_DEBUG}"

```

Comment:
On Windows this will put .lib files into the directory where the .exe files are.

Reply:
Not according to the cmake documentation.

https://cmake.org/cmake/help/latest/manual/cmake-buildsystem.7.html#library-output-artifacts

I'll retest on windows.

Reply:
Maybe it was like that in the past. If it works as desired now, then I'm fine with the change.